vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/treegrid/it/TreeGridBasicFeaturesPage.java

```diff
@@ -158,7 +154,7 @@ private void createHierarchyColumnSelect() {
         options.put("index", HierarchicalTestBean::getIndex);
 
         options.entrySet().forEach(entry -> {
-            addAction("Set hierarchy column - " + entry.getKey(), () -> {
+            addButton("set-hierarchy-column-" + entry.getKey(), () -> {
```

Comment:
Searching by an element ID is easier than arbitrary strings that are transformed to an id.